### DISCUSSION TITLE
Option to Switch Access to Different GitHub Account?

### DISCUSSION THREAD

**Discussion by ecodylan** [community/community#14013] [Copilot Conversations]
I just received access to copilot today and am very excited to try it. However, I had initially signed up for the waitlist with an email for a github account that is now outdated. I would prefer to use copilot with my updated/current account. Is it possible to swap my access from my old email to my new one?

Thank you :) :) :)

**Comment by bendhu** [comment]
my scenario is i will be using my company account for a while. Would it possible to access Copilot from my personal account? Thanks.

**Comment by vinsevinalon** [comment]
Any updates in this scenario? I need to switch account from my work account to personal account.
